### PR TITLE
Fix hero details columns

### LINKED ISSUE #623: Details section is not aligned
### Describe the bug

Details section is not aligned

### Screenshots

Screenshot

### PR DESCRIPTION
**Pull request by jendowns**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR adds a styled component called `GridColumns`, which allows you to pass in a number of columns for `grid-template-columns` & also uses `display: grid`. By default, without a `columns` prop value, `grid-template-columns` will just repeat 1 column.

I think this type of change would be useful to the "details" section of the `Hero` component. Since that part of the page has content with varying height, `display: grid` will help ensure that each "detail" section is aligned by making the content conform to the grid layout.

In summary:

### Added

- `GridColumns` styled component

### Changed

- Replaced `FloatLeft`, `FloatRight`, and `Flex` usage in the "details" section of the `Hero` component with the new `GridColumns` styled component
- Changed image styles for the "details" section from `float: left` to `align-self: flex-start`
  **NOTE:** I realize this is a little different than what you had before, and isn't quite related to #623, but without this style update, the images are vertically centered & I thought that didn't look quite right after making the grid changes described above. With this style change though, the images will all be positioned in the same location, regardless of how much text is in a "detail" section. But please let me know if you disagree & I'd be happy to change it back! 👍 

Screenshot:

Screen Shot 2020-06-11 at 7 58 16 PM

## Related Tickets & Documents

Resolves #623 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] Are there any post deployment tasks we need to perform?

No I don't think so. 👍 

## [optional] What gif best describes this PR or how it makes you feel?

BMO from Adventure Time is dancing.

### CODE REVIEW

File: src/styles/Header/Hero.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ const DetailsSection = styled(Hero)`
     text-align: left;
   }
   img {
-    float: left;
+    align-self: flex-start;
```
Comment (bdougie) [inline src/styles/Header/Hero.js:219]: Love it.

**Comment by bdougie** [comment]
@jendowns thanks so much for the quick PR on this. I also appreciate the CSS Grid addition as well. I attempted using Grid for this originally but not as familiar with it as I am with FlexBox. 

I am also a fan of the changelog copy you included in the PR!

**Comment by bdougie** [comment]
Before I merge, @yg can you take a look? Feel free to merge if you approve.

**Comment by yg** [comment]
**LGTM** :tada:, Thanks @jendowns for the PR!

**Pull request merged**